ccpp/data/GFS_typedefs.meta

```diff
@@ -2395,6 +2433,14 @@
   type = real
   kind = kind_phys
   active = (control_for_stochastic_land_surface_perturbation /= 0)
+[dqdti]
+  standard_name = instantaneous_water_vapor_specific_humidity_tendency_due_to_convection
```

Comment:
`tendency_of_specific_humidity_due_to_convection`; All variables are assumed instantaneous unless otherwise specified; there is no need for water vapor since it is inherent in "specific humidity"